ui/src/components/diagram/WorkflowGraph.jsx

```diff
@@ -338,8 +338,21 @@ class WorkflowGraph extends React.Component {
     inner.selectAll("g.node").on("click", this.handleClick);
   };
 
+  /**
+   * Get the taskRef id base on browsers
+   * @param e
+   * @returns {string | undefined} The id of the task ref
+   */
+  getTaskRef = (e) => {
+    const isFirefox = navigator.userAgent.toLowerCase().indexOf("firefox") > -1;
+    if (isFirefox) {
```

Comment:
This fix should also work in the latest version of Chrome:
![image](https://user-images.githubusercontent.com/1336974/212436618-d60ac236-72e8-49b1-9e14-26354acfc3f3.png)